package.json

```diff
@@ -12,21 +12,23 @@
     "type-check": "biome lint && tsc",
     "test": "pnpm --color -r run test",
     "ci:install": "pnpm install --no-frozen-lockfile",
-    "ci:version": "changeset version && pnpx @qingshaner/jsr-release --allowDirty version --sync",
+    "ci:version": "changeset version && pnpm add -D @pnpm/logger@^5.1.0 @pnpm/worker@^1000.1.3 && pnpx @qingshaner/jsr-release --allowDirty version --sync",
     "ci:publish": "changeset publish",
     "ci:prepublish": "pnpm build",
+    "deps:check": "pnpm dlx knip --production --no-exit-code --reporter=md",
```

Comment:
`--no-exit-code` means it'll always exit with a zero exit code (i.e. it won't fail)

do we actually want it to fail so CI won't publish if knip errors?

Reply:
i thought that it was going to fail if removed it 😅